prometheus_client/__init__.py

```diff
@@ -300,6 +304,73 @@ def _samples(self):
           ('_count', {}, self._count),
           ('_sum', {}, self._sum))
 
+def _floatToGoString(d):
+  if d == _INF:
+    return '+Inf'
+  elif d == _MINUS_INF:
+    return '-Inf'
+  else:
+    return repr(d)
+
+@_MetricWrapper
+class Histogram(object):
+  _type = 'histogram'
+  _reserved_labelnames = ['histogram']
+  def __init__(self, buckets=(.005, .01, .025, .05, .075, .1, .25, .5, .75, 1.0, 2.5, 5.0, 7.5, 10.0, _INF)):
```

Comment:
This .75 in there really gives me the creeps... I think the default buckets should have a more regular spacing.